### PR TITLE
Add JMX SSL setting

### PR DESCRIPTION
**Pull request by gquintana**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by gquintana** [comment]
See issue #274

**Comment by gquintana** [comment]
See https://blogs.oracle.com/jmxetc/entry/jmx_connecting_through_firewalls_using

File: app/views/cluster/updateCluster.scala.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 </div>
 }
 }
-
+K
```
Comment (patelh) [inline app/views/cluster/updateCluster.scala.html]: K?

File: app/kafka/manager/model/model.scala [hunk 2]
```diff
@@ -184,6 +187,7 @@ object ClusterConfig {
           val jmxEnabled = fieldExtended[Boolean]("jmxEnabled")(json)
           val jmxUser = fieldExtended[Option[String]]("jmxUser")(json)
           val jmxPass = fieldExtended[Option[String]]("jmxPass")(json)
+          val jmxSsl = field[Boolean]("jmxSsl")(json)
```
Comment (patelh) [inline app/kafka/manager/model/model.scala]: Please use field extended:
```
fieldExtended[Boolean]("jmxSsl")(json)
```

**Pull request merged**